crates/criticalup-core/src/download_server_client.rs

```diff
@@ -117,17 +118,28 @@ impl DownloadServerClient {
         // If the token contains such chars treat the authentication as failed due to an invalid
         // token, as the server wouldn't be able to validate it either anyway.
 
-        // set path to token file for docker
+        // Set path to token file for docker.
         let path_to_token_file = if std::path::Path::new("/.dockerenv").exists() {
             Some("/run/secrets/CRITICALUP_TOKEN")
         } else {
             None
         };
 
-        let header = self
+        // Check for token in
+        // 1. Docker
+        // 2. State file (included in call to `self.state.authentication_token(...)`)
```

Comment:
I think Docker > Env > State no?

Reply:
Ah! my bad, I assumed State > Env.

Reply:
Hmmmm if you do

```bash
criticalup auth set $TOKEN
```

Then run

```bash
CRITICALUP_TOKEN="blah" criticalup install
```

Would you expect it to go use the token from state over the one you specified for that exact command?